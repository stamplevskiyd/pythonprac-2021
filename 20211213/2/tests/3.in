[75, 65, 72, 97, 98, 95, 72, 84, 64, 16, 19, 56, 60, 8, 29, 2, 5, 61, 7, 31, 44, 40, 43, 61, 48, 0, 20, 51, 24, 76, 29, 81, 19, 56, 24, 90, 72, 32, 53, 77, 80, 84, 73, 49, 57, 98, 67, 88, 91, 24, 12, 86, 75, 78, 63, 13, 11, 3, 54, 63, 58, 43, 17]