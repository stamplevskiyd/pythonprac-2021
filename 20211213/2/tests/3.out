[0, 2, 3, 5, 7, 8, 11, 12, 13, 16, 17, 19, 19, 20, 24, 24, 24, 29, 29, 31, 32, 40, 43, 43, 44, 48, 49, 51, 53, 54, 56, 56, 57, 58, 60, 61, 61, 63, 63, 64, 65, 67, 72, 72, 72, 73, 75, 75, 76, 77, 78, 80, 81, 84, 84, 86, 88, 90, 91, 95, 97, 98, 98]
